lua/customroles/elementalist/shared.lua

```diff
@@ -125,7 +125,8 @@ ROLE.desc = [[You are an {role}! {comrades}
 
 Bullets you shoot may activate special effects when they hit your target.
 
-Press {menukey} to purchase new effects as you unlock additional equipment points!]]
+Press {menukey} to purchase new effects as you unlock additional credits!]]
+ROLE.shortdesc = "Can purchase elemental abilities and upgrades from their shop which cause different affects on the players they shoot."
```

Comment:
```suggestion
ROLE.shortdesc = "Can purchase elemental abilities from their shop which cause different effects on the players they shoot."
```
Again I think omitting the upgrade bit makes it a bit less wordy